djmoney/models/fields.py

```diff
@@ -354,10 +354,10 @@ def south_field_triple(self):
     def deconstruct(self):
         name, path, args, kwargs = super(MoneyField, self).deconstruct()
 
-        if self.default:
+        if self.default is not None:
             kwargs['default'] = self.default.amount
         if self.default_currency != DEFAULT_CURRENCY:
-            kwargs['default_currency'] = self.default_currency
+            kwargs['default_currency'] = '%s' % str(self.default_currency)
```

Comment:
`'%s' % str(self.default_currency)` should just be `str(self.default_currency)` ?